hack/functest.sh

```diff
@@ -43,3 +43,30 @@ TEST_NAMESPACE=node-maintenance-operator go test ./test/e2e/... -root=$(pwd) -ku
 
 echo "E2e tests passed"
 
+echo "check validation of openaAPIV3Schema"
+
+$KUBECTL_CMD create -f _out/namespace-init.yaml
+
+$KUBECTL_CMD create -f _out/nodemaintenance_crd.yaml
+
+echo "validate CRD"
+
+VALIDATE_CRD=$($KUBECTL_CMD get -o yaml crd nodemaintenances.kubevirt.io)
+if [[ $VALIDATE_CRD == "" ]]; then
+	echo "can't validate CRD, check if deployment is running"
+	exit 1
+fi
+
+set +e
+VALIDATION_ERRORS=$(echo "$VALIDATE_CRD" | grep -c "spec.validation.openAPIV3Schema")
+set -e
+
+if [[ $VALIDATION_ERRORS != "0" ]]; then
+	echo "validation of CRD failed"
```

Comment:
Should we display the errors?

Reply:
done

Reply:
Wouldn't we just want the contents of VALIDATION_ERRORS?